python_ta/transforms/ExprWrapper.py

```diff
@@ -1,9 +1,13 @@
-from typing import Dict, List, Optional, Union
+from typing import Any, Dict, List, Optional, Union
 
 import astroid
-import z3
 from astroid import nodes
 
+try:
```

Comment:
I did this because if `z3` were set to `Any` or `None` in some circumstances, referencing `z3.ExprRef` or any of `z3`'s other attributes would raise an error.

Reply:
Overall now that we've settled on the import strategy in the `graph.py` file, you should undo the changes you mentioned in this comment. More specifically:

1. This file (`ExprWrapper.py`) should *assume* that `z3` can always be imported correctly.
2. In `graph.py`, make the importing of `ExprWrapper` to be conditioned on whether the import of `z3` was successful. This would look like

```python
try:
    from z3 import ExprRef
    from ..transforms import ExprWrapper
except ImportError:
    ExprRef = Any
    ExprWrapper = Any
```